mindsdb/integrations/handlers/ray_serve_handler/ray_serve_handler.py

```diff
@@ -30,10 +30,16 @@ def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[
         args = args['using']  # ignore the rest of the problem definition
         args['target'] = target
         self.model_storage.json_set('args', args)
+        # TODO: use headers dynamically
+        headers = {
+            'Content-Type': 'application/json; format=pandas-records',
+        }
+        if 'headers' in args and 'authorization' in args['headers']:  #added header authorization key based condition
```

Comment:
The idea is to send the authorization in the USING clause as:
```sql
CREATE MODEL mindsdb.byom_ray_serve
FROM ..
PREDICT X
USING
    authorization = 'Bearer xyz'
```
If so, we don't need the `headers` or check for it here